convert/convert.go

```diff
@@ -194,6 +196,78 @@ func bedMidpoint(b bed.Bed) int {
 	return (b.ChromEnd + b.ChromStart) / 2
 }
 
```

Comment:
Add a large comment above this function explaining how it works (see some of the other functions in this file as a template). Note that the comment conventionally starts with a space and then the function name.